src/Lodge/Postcode/Postcode.php

```diff
@@ -136,4 +129,25 @@ public function lookup($postcode)
 		return $array;
 	}
 
+    private function callGoogleApi($url) 
+    {
+        $url = $this->addApiKeyToUrl($url);
+        $json = json_decode(file_get_contents($url));
+        $this->checkApiError($json);
+        return $json;
+    }
+
+    private function addApiKeyToUrl($url)
+    {
+        return ($api_key = \Config::get('postcode-lookup::apikey')) ?
```

Comment:
This has a hard dependency on Laravel. Instead of using `Config` here, can we pass it through the constructor? Some people are using this package outside Laravel, and this would be a breaking change.